scripts/convert-svgs-to-data-uri.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs');
+const path = require('path');
+const SVGO = require('svgo');
+
+const svgDir = './scripts/svgs/';
+const symbols = /[\r\n%#()<>?\[\\\]^`{|}]/g;
+
+const svgo = new SVGO();
+
+function encodeSVG(data) {
+  // Use single quotes instead of double to avoid encoding.
+  data = data.replace(/"/g, "'");
+
+  data = data.replace(/>\s{1,}</g, '><');
+  data = data.replace(/\s{2,}/g, ' ');
+  data = data.replace(symbols, encodeURIComponent);
+  data = data.replace(/fill='%23(.*)' /g, "fill='#{vf-url-friendly-color($replace-me-with-color-variable)}' ");
```

Comment:
Seems that we call default icon color variable `$color`, so maybe we can do it here as well right of the box?

Reply:
Yeah, I was in two minds about this, I wanted to make sure it was explicit to a developer that they have to do something here, especially if there's more than one color. That said, the first thing I did when adding these to the SCSS file was replace these, so yes, I will do this :smile: